go.mod

```diff
@@ -2,6 +2,9 @@ module github.com/cert-manager/cmctl/v2
 
 go 1.23.0
 
+// https://github.com/kubernetes/apiserver/issues/116
+replace github.com/google/cel-go => github.com/google/cel-go v0.22.1
```

Comment:
This will break installing cmctl using go, which does not support replace statements.